modules/actions.js

```diff
@@ -1,33 +1,28 @@
-class Data {
-  static getLocalStorage() {
-    let allScores;
-    if (localStorage.getItem('allScores') === null) {
-      allScores = [];
-    } else {
-      allScores = JSON.parse(localStorage.getItem('allScores'));
-    }
-    return allScores;
-  }
-}
+const api = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/:Fh0ZLhL9PMX1g7EtaJDa/scores';
 
+/* eslint-disable  quote-props */
 export default class Actions {
-  static display() {
-    const books = Data.getLocalStorage();
+  static addScore() {
+    const name = document.querySelector('#name').value;
+    const score = document.querySelector('#score').value;
```

Comment:
As requested by the project requirements:
![image](https://user-images.githubusercontent.com/42799579/202020327-a27f7651-19f0-48d8-bad8-0767c2f76d62.png)

Please use the ES6 syntax, with arrow functions. Kindly update the `addScore`, `clearValues`, `refreshAll` & `getScores` functions to the updated syntax 😄.

Reply:
@Gopxfs Hi. I was using Classes instead of regular functions for addScore and clearValues. Is that ok, or should I also change them to arrow functions?

Reply:
Hello, this is a tricky one 🤔, we shouldn't use arrow functions as methods. Although, since it is a requirement of the project (we need to check that you understand what are arrow functions), you should use them. I don't think this will give you problems with this project, but if something wrong happens, please consider going back to the methods 😉.

Reply:
@Gopxfs Ok, I changed the method to arrow functions. I was getting some import and export errors but I fixed them. Please let me know if I missed something.

Reply:
You can request a review, and another reviewer will check it 😃